src/fastoad/openmdao/tests/test_validity_checker.py

```diff
@@ -230,3 +233,27 @@ def setup(self):
     ValidityDomainChecker.log_records(records)
     with open(log_file_path) as log_file:
         assert len(log_file.readlines()) == 4
+
+
+def test_sellar(caplog):
```

Comment:
Another comment, which is rther a question. When I click on the little triangle to run only this tests in PyCharm, 3046 test are collected and ran. Although when I run the file hat contains this test, only 3 are collected. Do you have any idea why that might happen ?